resources/views/profile/edit.blade.php

```diff
@@ -0,0 +1,29 @@
+@extends('layouts.layout')
+
+@section('content')
+    <div name="header">
+        <h2 class="font-semibold text-xl text-gray-800 dark:text-gray-200 leading-tight">
+            {{ __('Profile') }}
+        </h2>
+    </div>
+
+    <div class="py-12">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8 space-y-6">
+            <div class="p-4 sm:p-8 bg-white dark:bg-gray-800 shadow sm:rounded-lg">
+                <div class="max-w-xl">
+                    @include('profile.partials.update-profile-information-form')
```

Comment:
Ik krijg geen foutmelding in beeld wanneer ik verkeerde informatie invul

Reply:
fixed